bot/cogs/LastFM.py

```diff
@@ -0,0 +1,215 @@
+        embed.add_field(name='Album',value=data['album']['title'])
+        embed.set_thumbnail(url=data['album']['image'][2]['#text'])
+        await ctx.send(embed=embed)
+      except IndexError:
+        await ctx.send('Invalid search term, try again')
+
+    @commands.command(
+      name="getalbum",
+      brief="gets info about an album",
+      description = "takes in album by itself or album followed by artist seperated by ; ",
+      aliases = ['album','getal']
+    )
+    async def getalbum(self,ctx,*,args):
+      try:
+        if '; ' in args:
+          args = args.split('; ')
+          data = get_album(args[0],args[1])
+        else:
+          data = get_album(args)
+        embed = discord.Embed(title=f"{data['album']['name']} by {data['album']['artist']}",url=data['album']['url'])
+        embed.add_field(name='Artist',value=data['album']['artist'])
+        embed.set_thumbnail(url=data['album']['image'][2]['#text'])
+        try:
+          embed.add_field(name='Release Data',value=data['album']['wiki']['published'],inline=False)
+        except KeyError:
+          pass
+        try:
+          embed.add_field(name='About',value=data['album']['wiki']['summary'].split('<a',1)[0])
+        except KeyError:
+          pass
+        await ctx.send(embed=embed)
+      except IndexError:
+        await ctx.send('Invalid search term, try again')
+      
+    @commands.command(
+      name = "getartist",
+      brief = "A command that get's info about an artist",
+      description = "Takes in just artist name",
+      aliases = ['artist','getar']
+    )
+    async def getartist(self,ctx,*,args):
+      try:
+        data = get_artist(args)
+        album_data = get_data('artist.gettopalbums&artist='+urllib.parse.quote(data['name']))['topalbums']['album']
+        bio = data['bio']['summary'].split('\n',2)[0].split('<a',1)[0]
+        if bio == '':
+          bio = data['bio']['summary'].split('\n',2)[1].split('<a',1)[0]
+        embed = discord.Embed(title=data['name'],url=data['url'],description=bio)
+        embed.set_thumbnail(url=album_data[0]['image'][2]['#text'])
+        top_albums = "\n".join([x["name"] for x in album_data[:10]])
+        similar = "\n".join([x["name"] for x in data["similar"]["artist"]])
+        if not similar:
+            similar = 'No Similar Artists Avaliable'
+        embed.add_field(name='Top Albums',value=top_albums,inline=True)
+        embed.add_field(name='Similar Artists',value=similar,inline=True)
+        await ctx.send(embed=embed)
+      except IndexError:
+        await ctx.send('Invalid search term, try again')
+
+    @commands.command(
+      name="topsongs",
+      brief="Gets a list of the top songs on the world charts",
+      aliases = ['songs','tops']
+    )
+    async def topsongs(self,ctx):
+      data = top_tracks()
+      embed = discord.Embed(title='Top 10 Tracks',url ='https://www.last.fm/charts')
+      for f in enumerate(data[:10],1):
```

Comment:
enumerate should be unpacked using tuple unpacking. I.e 
`for x, y in enumerate(range(10), range(10, 21))`

using indexes is messy and confusing. Also: var names should be descriptive. `f` should be descriptive to what it actually is.

an example could be `count, info in enumerate()`